### PR TITLE
Updated versioning to be correct with internal semantic versioning

### PR DESCRIPTION
**Pull request by digitalergonom**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: knuddels-userapps-api/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Knuddels UserApps API 1.0
+// Type definitions for Knuddels UserApps API 1.0.00093736
```
Comment (mhegazy) [inline knuddels-userapps-api/index.d.ts]: versions are only `major.minor`

Comment (shameister) [inline knuddels-userapps-api/index.d.ts]: Thank you. We will adjust this again.

**Comment by mhegazy** [comment]
The latest file seems to have this property set, see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/knuddels-userapps-api/index.d.ts#L1544

is this PR still needed?

**Comment by paulvanbrenk** [comment]
Closing this PR, as it seems the file in master has been updated.

**Pull request closed without merging**